packages/api/src/graphql/comment.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  GraphQLObjectType,
+  GraphQLNonNull,
+  GraphQLID,
+  GraphQLEnumType,
+  GraphQLInputObjectType,
+  GraphQLInt,
+  GraphQLList
+} from 'graphql'
+import {GraphQLDateTime} from 'graphql-iso-date'
+import {Context} from '../context'
+import {
+  CommentAuthorType,
+  CommentRejectionReason,
+  CommentRevision,
+  CommentStatus
+} from '../db/comment'
+import {GraphQLRichText} from './richText'
+
+export const GraphQLCommentStatus = new GraphQLEnumType({
+  name: 'CommentStatus',
+  values: {
+    Approved: {value: CommentStatus.Approved},
+    PendingApproval: {value: CommentStatus.PendingApproval},
+    PendingUserChanges: {value: CommentStatus.PendingUserChanges},
+    Rejected: {value: CommentStatus.Rejected}
+  }
+})
+
+export const GraphQLCommentRejectionReason = new GraphQLEnumType({
+  name: 'CommentRejectionReason',
+  values: {
+    Misconduct: {value: CommentRejectionReason.Misconduct},
+    Spam: {value: CommentRejectionReason.Spam}
+  }
+})
+
+export const GraphQLCommentAuthorType = new GraphQLEnumType({
+  name: 'CommentAuthorType',
+  values: {
+    [CommentAuthorType.Author]: {value: CommentAuthorType.Author},
```

Comment:
Is there a reason why you define the variables here in brackets `[ ]` ? I know that this should also work in this way but in order to have consistency with the rest of the code you might just write `Author:` ?

Reply:
for the sake of consistency only

Reply:
I made them all the same. what do you guys think?

Reply:
Not a fan of it. Use something more descriptive because that's also the enums being displayed in the API.